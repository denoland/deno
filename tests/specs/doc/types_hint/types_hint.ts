// @deno-types="./foo.d.ts"
export * from "./foo.js";
