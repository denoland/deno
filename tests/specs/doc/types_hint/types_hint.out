Defined in [WILDCARD]/foo.d.ts:2:14

const foo: string
  An exported value.

